mlflow/server/js/src/utils/Utils.js

```diff
@@ -170,11 +171,15 @@ class Utils {
   }
 
   /**
-   * Renders the run name into a string.
+   * Renders the run name into a string, truncating it if necessary.
    * @param runTags Object of tag name to MlflowMessages.RunTag instance
    */
   static getRunDisplayName(runTags, runUuid) {
-    return Utils.getRunName(runTags) || "Run " + runUuid;
+    const runName = Utils.getRunName(runTags) || "Run " + runUuid;
```

Comment:
I'm not sure we want to say "Run <whatever>" in the run list table, so maybe we can have another method that doesn't do that. It makes sense for the title of the Run page but it will look weird in the table by default to see "Run" in front of everything. It's very redundant.

Reply:
Makes sense - I've left run name blank for now if it's not set